### ISSUE TITLE
Parameters

### ISSUE THREAD

**Issue by edasque** [yoyostile/Plexpy-Ubersicht-Widget#1]
Can you clarify how to obtain the API key and what the host name is expected to be when you are not local.

**Comment by elads** [comment]
@edasque  I too am looking for clarification on the host name parameter. My widget simply displays the error "JSON Parse error: Unexpected EOF"

However I may be able to help with the API Key. 

once you have PlexPy installed, head to PlexPy's browser interface and go to Settings > Access Control. At the bottom of this page you can Enable API and generate a key. Press the save button and then allow PlexPy to restart. 

now, I haven't yet gotten this widget working and I have a feeling it has something to do with the host name parameter, but maybe this will help you take the next step. Good luck!

UPDATE: 
i got mine working with "PlexPyHost: 'localhost:8181'",  "protocol: 'http'" and the API key steps above. 
I realize that my host info may not be useful to you as you are trying to go remote. but at least I can verify the API Key process.

**Comment by edasque** [comment]
I got it 'working', as in I don't get an error message anymore. But nothing shows up.

**Comment by elads** [comment]
@edasque 
might be a simple thing, but nothing shows up for me unless something is actually playing or transcoding in plex. I used my iOS Plex app to watch a TV episode and it showed up in the widget appropriately.

**Comment by jgillman** [comment]
@elads There is a bug in the code. @InnerSpark has a fix in a Pull Request here: https://github.com/yoyostile/Plexpy-Ubersicht-Widget/pull/2

---

# TL;DR

In the coffee file, change the `plexpyHost` variable in the `command` line. It should be `PlexPyHost`. *Note the capitalization is different.*

**Comment by dv8godd** [comment]
Where were you 2 hours ago when this was driving me nuts?

I actually found the same error and submitted it separately before coming back to this thread.

**Comment by yoyostile** [comment]
Sorry, I had my notifications disabled for this repo :( I'll have a look at the PR now.

**Comment by yoyostile** [comment]
So everything should be fine again. :)
Excuse my late response!
